module_3/gifts/service/src/main/java/com/epam/esm/service/validation/TagValidation.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.esm.service.validation;
+
+import com.epam.esm.dto.TagDto;
+import com.epam.esm.exception.ErrorCode;
+import com.epam.esm.repository.model.EntityConstant;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import org.springframework.stereotype.Component;
+import org.springframework.util.MultiValueMap;
+
+@Component
+public class TagValidation {
+	private static final int MIN_NAME_LENGTH = 2;
```

Comment:
@LizavetaSinitsyna Try to create one file with all static consts and keep it there.